### ISSUE TITLE
[CoE Starter Kit - FEATURE] Clean Up Old Object App re-design

### ISSUE THREAD

**Issue by PPMini** [microsoft/coe-starter-kit#4302]
### Describe the issue

Once selected the App or Flow option from home screen (screen shot below). No way to navigate back.

<img width="1100" alt="image" src="https://user-images.githubusercontent.com/112712553/204565281-9cd9d843-5e72-4f7b-afa3-321cacd47427.png">

Next screen, but no way to go back to Home screen shown above

<img width="1157" alt="image" src="https://user-images.githubusercontent.com/112712553/204566340-54c6a7a2-dafe-4895-9fc6-bcdb0a2f2ffe.png">

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

November

### What app or flow are you having the issue with?

CleanUp Old Objects App

AB#1554

**Comment by manuelap-msft** [comment]
You can use the Hamburger menu in the top right corner to navigate to all other screens.

We have re-designing this app on the backlog so the app will get a new look and feel using [Creator Kit](https://aka.ms/creatorkit) components. We'll take this feedback on board for the redesign.

**Comment by PPMini** [comment]
Yes Manuela, using the Hamburger menu I can access all other screens but not the home screen. 
Thanks for the update.
